worker/worker.go

```diff
@@ -156,6 +156,7 @@ func (w *emailWorker) SendSimpleMessage(orgEmail string, msg *email.EmailMessage
 		}
 
 		action := func() error {
+			msg.From = orgEmail
```

Comment:
change the declaration of SendSimpleMessage(orgEmail string, msg *email.EmailMessage) to
SendSimpleMessage(msg *email.EmailMessage).